### ISSUE TITLE
Can't send images by pasting from clipboard in Arch Linux on Sway #699

### ISSUE THREAD

**Issue by kibybyte** [mirukana/mirage#229]
I use Arch Linux and Sway on hardware that fully supports it. I use grim, slurp and wl-copy to take screenshot, save it in my device as well as copy to my clipboard with the following command bound in sway configureation file:
```
#screenshots
bindsym Shift+Print exec grim -c -g "$(slurp)" -t png - | tee /home/$USER/pictures/ss/$(date +"%d-%b-%Y--%H:%M:%S").png | wl-copy
bindsym Print exec grim -c -t png - | tee /home/$USER/pictures/ss/$(date +"%d-%b-%Y--%H:%M:%S").png | wl-copy
```
In every other program I can just paste clipboard contents and send stuff except for mirage. 

Here is a sample of what happens. This just keeps rotating on and on. The program completely freezes if I forcibly click on send-

![image](https://i.ibb.co/Lk4Tws6/22-Jun-2021-20-39-32.png)

I have run from terminal also and gotten these outputs

![error1](https://i.ibb.co/K7KhMy4/22-Jun-2021-20-57-30.png)
![error2](https://i.ibb.co/C8nRT3r/22-Jun-2021-20-57-40.png)

However, If I try to send image by clicking from the UI, it works fine. Even PNG. From this button I mean -

![image](https://user-images.githubusercontent.com/85624218/122963045-8f2aaa00-d3a3-11eb-8747-135a5d2a5afc.png)

I love this project and how fast *and* beautiful it is. Kudos to the creators. 

TL;DR: Can't paste clipboard images into mirage running on arch linux on sway

**Comment by kibybyte** [comment]
BTW, I get same results regardless of whether I've installed matrix-mirage from the AUR or from the Chaotic AUR

**Comment by MRAAGH** [comment]
Possibly related to #195
